wger/exercises/static/js/graph.js

```diff
@@ -0,0 +1,72 @@
+/* jshint esversion: 6 */
+
+const Compare = () => {
+  let username = document.getElementById('username').value;
+  // set the dimensions and margins of the graph
+  const marginTop = 10;
+  const marginRight = 30;
+  const marginBottom = 20;
+  const marginLeft = 50;
+  let width = 460 - marginLeft - marginRight;
+  let height = 400 - marginTop - marginBottom;
+  // append the svg object to the body of the page
+  document.getElementById('my_dataviz').innerHTML = '';
+  let svg = d3.select('#my_dataviz')
+    .append('svg')
+    .attr('width', width + marginLeft + marginRight)
+    .attr('height', height + marginTop + marginBottom)
+    .append('g')
+    .attr('transform',
+      'translate(' + marginLeft + ',' + marginTop + ')');
+  // Parse the Data
+  d3.json('https://wger-kronos-pr-23.herokuapp.com/api/v2/get_weight_data/' + username, data => {
+    // List of subgroups = header of the csv files = soil condition here
```

Comment:
@Emmanuel-Dominic am wondering after merging this PR won't this feature fail because at the moment it is pointing, at the URL of this PR.

Reply:
This has been resolved, Thank you.